.github/workflows/_linux-test.yml

```diff
@@ -326,6 +326,14 @@ jobs:
           test_config: ${{ matrix.config }}
           job_identifier: ${{ github.workflow }}_${{ inputs.build-environment }}
 
+      - name: Upload the benchmark results
+        uses: pytorch/test-infra/.github/actions/upload-benchmark-results@upload-benchmark-results-with-additional-information
```

Comment:
I'll switch the branch here to main after https://github.com/pytorch/test-infra/pull/5918 lands